examples/hello-world/README.md

```diff
@@ -0,0 +1,235 @@
+  
+  <img src="/docs/images/info.png" height=25 /> Apache Karaf features file is deployed as Maven artifact using [Build Helper Maven Plugin](http://mojo.codehaus.org/build-helper-maven-plugin/). Usage of this plugin is defined in [the Maven parent POM file](../pom.xml#L98). 
+
+Building
+--------
+
+In order to build the Hello World project, follow these steps:
+
+* It is necessary to have at least Git client, Oracle Java SE 6, Maven 3 and a decent development machine.
+* Download latest Apache Karaf 3.0.x [here](https://karaf.apache.org/index/community/download.html) and extract it in any user's folder.
+* Using Git client, clone MQNaaS code into any user's folder:
+
+  ```
+  git clone https://github.com/dana-i2cat/mqnaas
+  ```
+
+* Build MQNaaS with:
+
+  ```
+  cd mqnaas/
+  mvn clean install
+  ```
+  
+  If everything worked as expected, this output should be something like:
+  
+  ```
+  ...
+  [INFO] BUILD SUCCESS
+  ...
+  ```
+
+* Execute Karaf as exposed [here](/README.md#executing).
+* Install Hello World Karaf feature with:
+  ```
+  feature:repo-add mvn:org.mqnaas.examples/hello-world/0.0.1-SNAPSHOT/xml/features
+  feature:install mqnaas-examples-helloworld
+  ```
+  
+  MQNaaS will be installed, since it is one of the dependencies.
+  
+  Once it is done, it is possible to check if everything worked (no errors are present in the shell). Moreover, this command:
+  
+  ```
+  list | grep "Hello World"
+  ```
+  
+  should return something like this:
+  
+  ```
+  176 | Active |  80 | 0.0.1.SNAPSHOT | MQNaaS :: Examples :: Hello World
+  ```
+  
+  * Test capability
+  
+  The way to test the Hello world capability is to create a resource with the capability and calling the auto-published REST API. To use `cURL` as HTTP client is the simplest option.
+  
+    * Create a resource with this command in a shell:
+    
+    ```
+    curl -i -X PUT -H "Content-Type: application/xml" -d '<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+    <ns2:rootResourceDescriptor xmlns:ns2="org.mqnaas">
+        <specification>
+            <type>OTHER</type>
+            <model>hello-world</model>
+            <version>1.0</version>
+        </specification>
+        <endpoints>
+          <endpoint />
+        </endpoints>
+    </ns2:rootResourceDescriptor>' http://localhost:9000/mqnaas/IRootResourceAdministration/
+    ```
+    
+    The expected response should be something like this:
+    
+    ```
+    HTTP/1.1 200 OK
+    Content-Type: application/octet-stream
+    Date: Tue, 14 Apr 2015 13:26:24 GMT
+    Content-Length: 23
+    Server: Jetty(8.1.15.v20140411)
+    
+    Other-hello-world-1.0-2
+    ```
+    
```

Comment:
Should we explain that the response body is the resource id? or will it be explained in the IRootResourceAdministration capability documentation?